crates/ruff/src/message/azure.rs

```diff
@@ -19,10 +19,7 @@ impl Emitter for AzureEmitter {
             let location = if context.is_jupyter_notebook(message.filename()) {
                 // We can't give a reasonable location for the structured formats,
                 // so we show one that's clearly a fallback
-                SourceLocation {
```

Comment:
I think we could use ::default here instead of min. That would also allow to use unwrap_or_default in many places